### ISSUE TITLE
Copy & paste introduces nbsp `\u00a0` character

### ISSUE THREAD

**Issue by themarcelor** [shomali11/slacker#94]
When someone copies & pastes commands, they end up with a weird character `\u00a0` do the command ends up being ignored.
> U+00A0 No-Break Space (NBSP) Unicode Character).

e.g., The payload shows something like "text":"\u00a0my-command arg1 arg2"

The payload can be visualized if we start Slacker in DEBUG mode:
```
                bot = slacker.NewClient(slackBotToken, slackAppToken, slacker.WithDebug(true))
```

Perhaps the input could be sanitized?

**Comment by themarcelor** [comment]
Created a PR 🤷🏼 
https://github.com/shomali11/slacker/pull/95

**Comment by arusso** [comment]
Closed with the merging of #95
